roles/openshift_health_checker/openshift_checks/logging/logging.py

```diff
@@ -51,15 +51,16 @@ def get_pods_for_component(self, execute_module, namespace, logging_component, t
 
     @staticmethod
     def not_running_pods(pods):
-        """Returns: list of pods not in a ready and running state"""
+        """Returns: list of pods not in a ready and running state
+        or an OpenShiftCheckException if no pods have been scheduled."""
```

Comment:
> doesn't throw an exception now :)

:/ saw this as I started to make changes and knew I would somehow forget to edit it out before pushing